src/tests/ftest/util/daos_utils.py

```diff
@@ -255,15 +250,21 @@ def pool_list_cont(self, pool, svc, sys_name=None):
             sys_name (str, optional): System name. Defaults to None.
 
         Returns:
-            CmdResult: Object that contains exit status, stdout, and other
-                information.
+            dict: Dictionary that contains the list of UUIDs in the key "uuids".
 
         Raises:
             CommandFailure: if the daos pool list-containers command fails.
 
         """
-        return self._get_result(
+        self._get_result(
             ("pool", "list-containers"), pool=pool, svc=svc, sys_name=sys_name)
+        # Sample output.
+        # c8bfc7c9-cb19-4574-bae2-af4046d24b58
+        # 182347e4-08ce-4069-b5e2-0dd04406dffd
+        data = {}
+        match = re.findall(r"([0-9a-f-]{36})", self.result.stdout)
+        data["uuids"] = match
+        return data
```

Comment:
Since this is one is simple, why not save a few lines?
```suggestion
        return {"uuids": re.findall(r"([0-9a-f-]{36})", self.result.stdout)}
```

Alternatively, if there is a concern that a failed command's stdout could match the regex (include a UUID) then we could ensure that the returned dictionary was empty in this case with:
```suggestion
        data = {}
        if self.result.exit_status == 0:
            data["uuids"] = re.findall(r"([0-9a-f-]{36})", self.result.stdout)
        return data
```

Reply:
Done. Thanks for suggestion.